project-base/app/src/FrontendApi/Resolver/Products/ProductsQuery.php

```diff
@@ -160,6 +180,34 @@ public function productsQuery(Argument $argument)
             $argument,
         );
 
+        if ($argument['categorySlug']) {
+            $category = $this->categoryQuery->categoryOrSeoMixByUuidOrUrlSlugQuery($info, urlSlug: $argument['categorySlug']);
```

Comment:
```suggestion
            $category = $this->categoryQuery->categoryOrSeoMixByUuidOrUrlSlugQuery($info, $argument['categorySlug']);
```

Reply:
second argument is UUID, not slug...

Reply:
After discussing the changes, I removed the named parameter and included `null` for the UUID.